Xamarin.RangeSlider.Droid/RangeSliderControl.cs

```diff
@@ -224,6 +224,15 @@ public bool MaxThumbHidden
 
         public Bitmap ThumbDisabledImage { get; set; }
         public Bitmap ThumbImage { get; set; }
+        private Bitmap _customThumbImage;
+        public Bitmap CustomThumbImage 
```

Comment:
I don't like the idea of having three different properties for each image. It's better to split `UpdateThumbImages` into two functions.